src/config/database.ts

```diff
@@ -3,13 +3,15 @@ import dotenv from "dotenv";
 dotenv.config();
 
 export const sequelize = new Sequelize(
-    "eCommerceTap",
-    process.env.DB_USERNAME as string,
-    process.env.DB_PASSWORD,
-    {
-      host: process.env.DB_HOSTNAME,
-      dialect: process.env.DB_DIALECT as 'mysql' | 'postgres' | 'sqlite' | 'mariadb' | 'mssql',
-      port: Number(process.env.DB_PORT),
-    }
-  );
+  "ecommercetap",
```

Comment:
database name could be also in env variable like the others

Reply:
It's how Mohammad did it, and I didn't change it.